packages/webcert/src/feature/list/List.tsx

```diff
@@ -23,6 +23,16 @@ interface Props {
   type: ListType
 }
 
+/**
+ * Generates a table with filtering options and sorting.
+ * @param icon specify what icon to use other vice undefined.
+ * @param config for this and sub-components
+ * @param list a list of all values that rows that should be displayed
+ * @param filter all filter options that can be applied in the above list
+ * @param title titletext
+ * @param type what type of page this is on
+ * @constructor
```

Comment:
I don't believe `@constructor` is valid here, since we are not returning a class.  @param should have nested properties, so I would expect for example `@param properties.icon ...` and `@param properties.config ...`.

Reply:
Updated comments and added refactoring in new committ.